tests/util.py

```diff
@@ -34,13 +34,11 @@ def assert_blank(r):
     Check that the fields that should be empty in a
     blank record are empty.
     """
-    assert r.records[0].baseid
-    assert r.records[0].did
-    assert not r.records[0].size
-    assert not r.records[0].acl
-    assert not r.records[0].authz
-    assert not r.records[0].hashes.crc
-    assert not r.records[0].hashes.md5
-    assert not r.records[0].hashes.sha
-    assert not r.records[0].hashes.sha256
-    assert not r.records[0].hashes.sha512
+    assert r["records"][0]["baseid"]
+    assert r["records"][0]["did"]
+    assert not r["records"][0]["size"]
+    assert not r["records"][0]["acl"]
+    assert not r["records"][0]["authz"]
+    assert not r["records"][0]["hashes"]
```

Comment:
the blank record used to have this (master branch):
```
'hashes': {'crc': None,
            'etag': None,
            'md5': None,
            'sha': None,
            'sha256': None,
            'sha512': None}
```
and now it has `'hashes': {}`
I think maybe because of the way you GET it? can you check what's going on?

Reply:
I changed this because the response of the GET leads to a key error when the keys within the hashes dictionary don't exist and I thought that the individual key assertions would be subsumed by the empty hash assertion, although I may be wrong about this and could find another work around if so.

Reply:
but why did the response contain all the keys before and not now?

Reply:
Note on this: looks like the swagger client was creating default "None" values for each hash type